Jenkinsfile

```diff
@@ -5,7 +5,7 @@ def slackChannel = '#devops-framework'
 
 pipeline {
     agent {
-        label 'codeontaplatest'
```

Comment:
@roleyfoley you mentioned previously that this was hardcoded elsewhere and so could not yet be updated. Just double-checking that this is no longer the case?